features/e2e_tests/e2e_card_payment_with_request_types.feature

```diff
@@ -0,0 +1,23 @@
+Feature: E2E Card Payments with request types in config
+
+  As a user
+  I want to use card payments method with request types config
+  In order to check full payment functionality
+
+  @reactJS
+  Scenario: Successful payment with config's requestTypes param having values in valid order
+    Given JS library is configured with VISA_CHECKOUT_WITH_REQUEST_TYPES_CONFIG and BASE_JWT
```

Comment:
Why VISA_CHECKOUT_WITH_REQUEST_TYPES_CONFIG if in scenario you use card payments method?